src/Components/Facility/FacilityCreate.tsx

```diff
@@ -470,7 +469,7 @@ export const FacilityCreate = (props: FacilityProps) => {
           Notification.Success({
             msg: "Facility updated successfully",
           });
-          navigate(`/facility/${facilityId}`);
+          // navigate(`/facility/${facilityId}`);
```

Comment:
commenting this line would stop it from redirecting to facility page on successful update, we will need this

Reply:
Sorry, I was checking something and forgot to uncomment it.